tasks/src/test/java/taf/page/YandexDiskDocumentPage.java

```diff
@@ -0,0 +1,65 @@
+package taf.page;
+
+import static taf.browser.Browser.waitElementToBeClickable;
+import static taf.constants.Constants.*;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+
+public class YandexDiskDocumentPage extends YandexDiskAbstractPage {
+
+  public YandexDiskDocumentPage(WebDriver driver) {
+    super(driver);
+  }
+
+  public YandexDiskDocumentPage switchToFrame() {
+    driver.switchTo().frame(driver.findElement(By.className("editor-doc__iframe")));
+    return this;
+  }
+
+  public YandexDiskDocumentPage typeTextInDocument(String text) {
+    waitElementToBeClickable(driver, TEXT_INPUT);
+    driver.findElement(TEXT_INPUT).sendKeys(text);
+    return this;
+  }
+
+  public YandexDiskDocumentPage saveStatusDocument() {
+    waitElementToBeClickable(driver, DOCUMENT_SAVE_STATUS);
+    driver.findElement(DOCUMENT_SAVE_STATUS);
+    return this;
+  }
+
+  public YandexDiskDocumentPage clickMenuItemFile() {
+    waitElementToBeClickable(driver, MENU_ITEM_FILE);
+    driver.findElement(MENU_ITEM_FILE).click();
+    return this;
+  }
+
+  public YandexDiskDocumentPage clickRenameButton() {
+    waitElementToBeClickable(driver, MENU_ITEM_SAVE_AS);
```

Comment:
Return a WebElement and your code will be shorter and faster:
waitElementToBeClickable(driver, MENU_ITEM_SAVE_AS).click();